.npmignore

```diff
@@ -1,8 +1,3 @@
-.env
-.DS_Store
-node_modules
-public/static.json
-public/bundles
-sam.json
-sam.yaml
-package-lock.json
+.github/
+scripts/
```

Comment:
If this is npm ignored then the postinstall script will fail which will not create the manifest.json that the CLI depends upon.

https://github.com/enhance-dev/enhance-starter-project/blob/main/package.json#L5

Reply:
Just making sure I understand: consumers run scripts/createManifest? Even though manifest.json is created in CI and already exists in the package tarball?

Reply:
It seems like I misunderstood the purpose of manifest.json and confused it with the vendor script in the other package.
so, good catch, this would break when npm installed.

Reply:
Now, that I've been thinking about it over a cup of tea we still may want to do the above. The whole purpose of the `createManifest` script was this project was very noisy with files a starter project didn't need. After these changes what would be in the npm package?